Topic5/src/Demo.java

```diff
@@ -0,0 +1,131 @@
+import java.sql.SQLException;
+import java.util.Scanner;
+
+public class Demo {
+
+	public static void main(String[] args) {
+		
+		java.sql.Connection connection;
+		try {
+			Class.forName("com.mysql.jdbc.Driver").newInstance();

```

Comment:
You could create a connector class to handle the connection in an independent class